habit/package.json

```diff
@@ -10,9 +10,18 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.10",
```

Comment:
where are we using this library?

Reply:
React navigation is dependent on masked-view

Reply:
I don't think so

Reply:
I don't think so